fix(audit-log): Add replay job how tos (#2732)
Kong/developer.konghq.com: app/konnect-platform/audit-logs.md

    description: "The HTTP response code; for example, `200` or `403`."
{% endtable %}
<!--vale on-->

## Recover audit logs

You can use replay jobs in {{site.konnect_short_name}} to recover audit logs. These are useful when you've missed audit log entries due to an error or a misconfigured audit log webhook.

> You can use either the {{site.konnect_short_name}} UI or API to configure a replay job.

Reviewer: ```suggestion
You can use either the {{site.konnect_short_name}} UI or the {{site.konnect_short_name}} API to configure a replay job.
```
- added [the {{site.konnect_short_name}}] for:
parallelism and super tight ownership for greater clarity

### {{site.konnect_short_name}} replay job

{% navtabs "replay-job" %}
{% navtab "UI" %}

> 1. In the {{site.konnect_short_name}} UI, click [**Organization**](https://cloud.konghq.com/organization) in the sidebar.

Reviewer: ```suggestion
1. In the {{site.konnect_short_name}} UI, from the sidebar, click [**Organization**](https://cloud.konghq.com/organization).
```

> 1. Click **Audit Logs Setup** in the sidebar.

Reviewer: ```suggestion
1. Click **Audit Logs Setup**..
```

   {% navtabs "portal-konnect" %}
   {% navtab "Konnect" %}
   1. Click the **Konnect** tab.
   1. Navigate to the region you want to configure the replay job for.
   1. Click the **Replay** tab.
   1. Select a time frame from the **Replay Time Range** dropdown menu.
   1. Click **Send Replay**.
   {% endnavtab %}
   {% navtab "Dev Portal" %}
   1. Click the **Dev Portal** tab.
   1. Click the Dev Portal you want to configure the replay job for.
   1. Click the **Replay** tab.
   1. Select a time frame from the **Replay Time Range** dropdown menu.
   1. Click **Send Replay**.
   {% endnavtab %}
   {% endnavtabs %}
{% endnavtab %}
{% navtab "API" %}
Send a `PUT` request to the `/audit-log-replay-job` endpoint:
<!--vale off-->
{% konnect_api_request %}
url: /v2/audit-log-replay-job
status_code: 201
method: PUT
body:
  start_at: $UTF_START_TIME
  end_at: $UTF_END_TIME
{% endkonnect_api_request %}
<!--vale on-->

{:.info}
> **Note:** The replay job is always sent to the webhook that is currently configured for the organization or Dev Portal at the time the replay job is executed. There is one webhook configuration per region.
{% endnavtab %}
{% endnavtabs %}

### Replay job status

> Once you configure a replay job, it will display the status. 

Reviewer: ```suggestion
Once you configure a replay job, it displays one of the following statuses. 
```
This as a section reads weird to me = this sentence (_Once you configure a replay job, it will display the status._) originally makes it seem like users will only be able to SEE the status of a reply job, AFTER they've configured it. But that doesn't make sense, since we have an [unconfigured] status.

A replay job can be in one of the following statuses:

<!--vale off-->
{% table %}
columns:
  - title: Status
    key: status
  - title: Description
    key: description
rows:
  - status: "Unconfigured"

>     description: Initial state. The job has not been set up.

Reviewer: ```suggestion
    description: The job has not been set up. This is the job's initial state.
```
